collector/src/bin/collector.rs

```diff
@@ -628,6 +630,18 @@ enum Commands {
         #[command(flatten)]
         db: DbOption,
     },
+
+    /// Displays diff between two local bench result.
+    BenchLocalDiff {
```

Comment:
Local should not be needed, as we're just comparing whatever is in the database. I would probably call this `BenchCmp`, `collector bench_cmp` sounds intuitive, as we want to compare two artifacts. The same name is also used for the compare page on the web.